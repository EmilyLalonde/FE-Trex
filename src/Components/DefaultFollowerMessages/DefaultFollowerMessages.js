import React, { Component } from "react";
import FollowerHeader from '../FollowerHeader/FollowerHeader';
import FollowerFooter from '../FollowerFooter/FollowerFooter'
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
  TextInput,
  Keyboard
} from "react-native";

export default class DefaultFollowerMessages extends Component {
  constructor() {
    super();
    this.state = {};
  }
  render() {
    const {navigate} = this.props.navigation;
    return (
      <View style={styles.container}>
        <FollowerHeader />
        <ScrollView>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 1</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 2</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 3</Text>
        </TouchableOpacity>
        <TouchableOpacity>
          <Text style={styles.button}>Default Message 4</Text>
        </TouchableOpacity>
        </ScrollView>
        <FollowerFooter navigate={navigate}/>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
    alignItems: 'stretch',
    justifyContent: 'flex-start'
  },
  button: {
    borderColor: "white",
    borderWidth: 1,
    borderRadius: 8,
    borderStyle: "solid",
    width: "auto",
    height: 60,
    margin: 20,
    fontSize: 30,
    padding: 10,
    color: "white",
    textAlign: "center",
    backgroundColor: "#84183B"
  }
});