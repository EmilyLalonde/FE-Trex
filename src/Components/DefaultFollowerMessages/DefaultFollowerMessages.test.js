import React, {Component} from 'react';
import { DefaultFollowerMessages } from './DefaultFollowerMessages';

describe('DefaultFollowerMessages', () => {
  it('should return true', () => {
    expect(true).toEqual(true)
  })
})