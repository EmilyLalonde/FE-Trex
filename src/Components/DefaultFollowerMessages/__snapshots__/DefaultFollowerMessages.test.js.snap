// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultFollowerMessages should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Header />
  <ScrollViewMock>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#84183B",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 1
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#84183B",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 2
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#84183B",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 3
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
    >
      <Text
        style={
          Object {
            "backgroundColor": "#84183B",
            "borderColor": "white",
            "borderRadius": 8,
            "borderStyle": "solid",
            "borderWidth": 1,
            "color": "white",
            "fontSize": 30,
            "height": 60,
            "margin": 20,
            "padding": 10,
            "textAlign": "center",
            "width": "auto",
          }
        }
      >
        Default Message 4
      </Text>
    </TouchableOpacity>
  </ScrollViewMock>
  <Footer
    navigate={[MockFunction]}
  />
</View>
`;
